bikesharing/migrations/0027_bike_non_static_bike_uuid.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.10 on 2020-06-10 16:35
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bikesharing', '0026_auto_20200605_1839'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='bike',
+            name='non_static_bike_uuid',
+            field=models.UUIDField(default=uuid.uuid4, help_text="A temporary ID used in public APIs, rotating it's value after each rent to protect users privacy.", null=True),
```

Comment:
can we add an unique-constraint for this field?

Reply:
ah, migration 0029

Reply:
yep, this is the recommended hack for prefilled fields. create, fill, make unique